plat/socionext/synquacer/drivers/scmi/sq_scmi.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2019, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <assert.h>
+
+#include <arch_helpers.h>
+#include <common/debug.h>
+
+#include <drivers/arm/css/scmi.h>
+#include <drivers/arm/css/css_mhu_doorbell.h>
+#include <plat/arm/common/plat_arm.h>
+#include <plat/common/platform.h>
+
+#include "scmi_sq.h"
+
+#include <sq_common.h>
+
+const uint32_t plat_css_core_pos_to_scmi_dmn_id_map[PLATFORM_CORE_COUNT] = {
```

Comment:
I think all the contents of this file can move to  sq_pm_scmi.c

This array can just be called `sq_core_pos_to_scmi_dmn_id[]`